tests/Integration/Behaviour/Features/Scenario/Product/Shop/shop_management.feature

```diff
@@ -57,13 +58,13 @@ Feature: Copy product from shop to shop.
     Then product productWithPrices is associated to shop shop1
     And default shop for product productWithPrices is shop1
     When I update product "productWithPrices" with following values:
-      | price              | 100.99          |
-      | ecotax             | 0               |
-      | tax rules group    | US-AL Rate (4%) |
-      | on_sale            | true            |
-      | wholesale_price    | 70              |
-      | unit_price         | 10              |
-      | unity              | bag of ten      |
+      | price           | 100.99          |
```

Comment:
these changes are not related to the PR, its just some style fixes (removes sume unnecessary spaces) that my IDE applied by autoformatting